packages/client/src/ConfigTest.ts

```diff
@@ -20,23 +21,25 @@ const sideChainConfig = {
  */
 export const CONFIG_TEST: StreamrClientConfig = {
     network: {
-        trackers: [
-            {
-                id: '0xb9e7cEBF7b03AE26458E32a059488386b05798e8',
-                ws: `ws://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30301`,
-                http: `http://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30301`
-            }, {
-                id: '0x0540A3e144cdD81F402e7772C76a5808B71d2d30',
-                ws: `ws://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30302`,
-                http: `http://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30302`
-            }, {
-                id: '0xf2C195bE194a2C91e93Eacb1d6d55a00552a85E2',
-                ws: `ws://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30303`,
-                http: `http://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30303`
-            }
-        ],
-        webrtcDisallowPrivateAddresses: false,
-        iceServers: []
+        layer0: {
+            entryPoints: [{
+                kademliaId: 'entryPointBroker',
+                type: 0,
+                websocket: {
+                    ip: '127.0.0.1',
+                    port: 40401
+                }
+            }],
+            peerDescriptor: {
+                kademliaId: uuid(),
```

Comment:
```suggestion
                kademliaId: 'd1cb4766-e44a-4b44-9e79-a7b62095c130',
```

Reply:
The DhtNode now (temporarily) generates a UUID for the PeerDescriptor if a PeerDescriptor is not specified by the user.